tests/monit/test_monit_status.py

```diff
@@ -31,13 +31,34 @@ def disable_lldp(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
         None.
     """
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
-    duthost.command("docker exec lldp supervisorctl stop lldpmgrd")
+
+    logger.info("Stopping 'lldpmgrd' process in 'lldp' container ...")
```

Comment:
Just a double confirm, do we need to run this on multi_asic DUT? I don't have a multi_asic device to verify.

Reply:
Yes, I think we need stop `lldpmgrd` process in `lldp` container which is running on Host and on each ASIC to check the format of Monit alerting message. Let me test this change on Multi-ASIC device again to verify.

Reply:
Bing, I tested this change on DuT `str-n3164-acs-2` again and it passed.